db/db.go

```diff
@@ -3,8 +3,6 @@ package db
 import (
 	"database/sql"
 	"log"
-
-	_ "github.com/lib/pq"
```

Comment:
Uhmmm. One important thing about Golang, the standard library is very powerful! So, for now, I recommend you to use it as much as you can. It will save you a lot of time and effort. Even for more complex projects you can use it. It's very well-documented and easy to use.

Reply:
The same here, for the gorilla/mux library, I think that the net/http library is more than enough for this project. But we can change this in the next step.

Reply:
Oh, got it, here i used gorilla/mux lib just for the sake of learning how it works.